src/PresentationalComponents/Inventory/Inventory.js

```diff
@@ -266,39 +267,53 @@ const Inventory = ({
     setDisableRuleModalOpen(disableRuleModalOpen);
   };
 
-  const createColumns = (defaultColumns) => {
-    let lastSeenColumn = defaultColumns.filter(({ key }) => key === 'updated');
-    let displayName = defaultColumns.filter(
-      ({ key }) => key === 'display_name'
-    );
-    let systemProfile = defaultColumns.filter(
-      ({ key }) => key === 'system_profile'
-    );
-    let tags = defaultColumns.filter(({ key }) => key === 'tags');
-
-    displayName = {
-      ...displayName[0],
-      transforms: [sortable, wrappable],
-      props: { isStatic: true },
-    };
-
-    lastSeenColumn = {
-      ...lastSeenColumn[0],
-      transforms: [sortable, wrappable],
-      props: { width: 20 },
-    };
+  const createColumns = useCallback(
+    (defaultColumns) => {
+      let lastSeenColumn = defaultColumns.filter(
+        ({ key }) => key === 'updated'
+      );
+      let displayName = defaultColumns.filter(
+        ({ key }) => key === 'display_name'
+      );
+      let systemProfile = defaultColumns.filter(
+        ({ key }) => key === 'system_profile'
+      );
+      let tags = defaultColumns.filter(({ key }) => key === 'tags');
+
+      displayName = {
+        ...displayName[0],
+        transforms: [sortable, wrappable],
+        props: { isStatic: true },
+        renderFunc: (name, id) => {
```

Comment:
We might want to only pass in a `renderFunc` if there is a `rule` set, otherwise it might link to somewhere unexpected.